src/components/lineChart/hooks/useRegionalSewerWaterChartOptions.ts

```diff
@@ -171,7 +172,7 @@ export function useRegionalSewerWaterChartOptions(
           description: replaceVariablesInText(text.daily_label_text, {
             name: selectedRWZI,
           }),
-          color: '#004277',
+          color: colors.chart.blueDark,
```

Comment:
If we already have multiple levels of nesting I think the camel casing could as well be replaced by an extra level. Might read a bit nicer. 
```suggestion
          color: colors.chart.blue.dark,
```

Reply:
Not sure if the chart sub-level is preferable. I don't expect the chart to have completely different blues from the other components. For now it's fine. We can also change it once we start duplicating colors.

Reply:
If we already have multiple levels of nesting I think the camel casing could as well be replaced by an extra level. Might read a bit nicer. 
```suggestion
          color: colors.chart.blue.dark,
```

Not sure if the chart sub-level is preferable. I don't expect the chart to have completely different blues from the other components. For now it's fine. We can also change it once we start duplicating colors.